### PR TITLE
Feature/191269 change grant comms

### PR DESCRIPTION
**Pull request by mshakirdfe**
Resolves #<1234>

## Type of change
https://educationgovuk-my.sharepoint.com/:w:/g/personal/muhammad_shakir_education_gov_uk/EZvpP3ie5sNAv9Jvg_260xMBfr3sYNHCmvy36dq6qVQ0kQ?e=gPkvRj

Azure DevOps 
https://dev.azure.com/dfe-gov-uk/Academies-and-Free-Schools-SIP/_workitems/edit/191260

### CODE REVIEW

**Review by dangood84 — DISMISSED** [review]

**Review by dangood84 — APPROVED** [review]

File: Dfe.Academies.External.Web/appsettings.Development.json [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+	"local_development": false,
+	"DetailedErrors": true,
+	"Logging": {
+		"LogLevel": {
+			"Default": "Information",
+			"Microsoft.AspNetCore": "Warning"
+		}
+	},
+	"academies_api": {
+		"endpoint": "https://webapp-t1dv-sip-a2c.azurewebsites.net/"
```
Comment (DrizzlyOwl) [inline Dfe.Academies.External.Web/appsettings.Development.json:11]: Seeing as though this and the `academisation_api:endpoint` values are duds, can we replace these URLs with just `"replace-me"` instead?

**Review by DrizzlyOwl — COMMENTED** [review]
I can't comment on the code, but I had a minor comment on the appsettings. Otherwise is fine

File: Dfe.Academies.External.Web/appsettings.Development.json (hunk 1 above)
Comment (mshakirdfe) [reply Dfe.Academies.External.Web/appsettings.Development.json]: Done - Don't worry about code changes

**Review by dangood84 — APPROVED** [review]

**Review by DrizzlyOwl — APPROVED** [review]
Looks good. Would be better if we can improve the code coverage as suggested by Sonar :)

**Pull request merged**